keras_cv/models/classification/classifier.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Image classifier model that works for most use cases."""
+
+import copy
+
+from keras import layers
+from tensorflow import keras
+
+from keras_cv.models.classification.classifier_presets import backbone_presets
+from keras_cv.models.classification.classifier_presets import (
+    backbone_presets_with_weights,
+)
+from keras_cv.models.classification.classifier_presets import classifier_presets
+from keras_cv.models.task import Task
+from keras_cv.utils.python_utils import classproperty
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class Classifier(Task):
+    """Image classifier with pooling and dense layers.
+
+    Args:
+        backbone: `keras_cv.models.Backbone`, the backbone architecture of the
+            classifier called on the inputs
+        num_classes: number of classes to predict
```

Comment:
nit: I think we're moving toward including types in docstrings, right?

So "integer, number of classes"